### DISCUSSION TITLE
Official Instance List on website

### DISCUSSION THREAD

**Discussion by trymeouteh** [YOURLS/YOURLS#3705] [Ideas]
Please create an official list of known yourls instances on the website for others to find instances to use.

**Comment by ozh** [comment]
I don't understand what you're asking for. Known public interfaces of YOURLS ? That would be a nightmare to maintain. Also, what would be the use?

**Comment by trymeouteh** [comment]
> I don't understand what you're asking for. Known public interfaces of YOURLS ?

Even a simple section on the github readme and users can submit issues to add/remove instances from the readme

> That would be a nightmare to maintain. 

Does not need to be up to date 24/7, even if it was updated once every few months is better than no list at all

> Also, what would be the use?

To easily find known public instances to use?

**Comment by LeoColomb** [comment]
Thanks for your input @trymeouteh.
I must say that I second @ozh.
Not only the maintenance cost is non-negligible, but the boundaries with advertisement* are then very thin.
And how would we moderate it? Should we accept public instances that promote violence in a way? What about political ones?
The value this would bring to the community is very little. Please keep in mind that you are asking volunteer people, who brought you free software in their spare time, to promote how their work is being used with very (very very) little return. 

That being said, you are very welcome to start such a list/repo/website and maintain in on your own. We'll be happy to add a link to your work on https://github.com/YOURLS/awesome.

\*Advertisement can already be done more or less for valuable [sponsors](https://github.com/sponsors/YOURLS).

**Comment by unwaivering** [comment]
How am I supposed to find an instance then? As a prospective user, who would like to use your software, how in  the world do you expect me to find one, if you don't keep a list somewhere?

**Reply by dgw** [reply]
Short answer: You are meant to _create_ an instance, not to _find_ one.

----

Long answer: YOURLS is a piece of software, not a service. Prospective users of YOURLS are _people who would host an instance_, because they want to have total control over link shortening for their project/brand/organization.

If you just want a link shortener, there are many _many_ services out there that'll take your long link and give back a shorter one. None of the entities running those services is necessarily any more or less reliable in the long term than whoever might run any random instance you would choose off of an "official" list, if we had one.
